pkg/ssh/ssh.go

```diff
@@ -77,42 +79,45 @@ type SSHTunnel struct {
 	client  *ssh.Client
 }
 
-func (s *SSHTunnel) copyBytes(out io.Writer, in io.Reader) {
+func (s *TheTunnel) copyBytes(out io.Writer, in io.Reader) {
 	if _, err := io.Copy(out, in); err != nil {
 		glog.Errorf("Error in SSH tunnel: %v", err)
 	}
 }
 
-func NewSSHTunnel(user, keyfile, host string) (*SSHTunnel, error) {
+// NewSSHTunnel create a new ssh tunnel for the given config with PrivateKey file.
+func NewSSHTunnel(user, keyfile, host string) (*TheTunnel, error) {
```

Comment:
If we change the name of the type from SSHTunnel then we should change the New function to match.

Reply:
Yes, I will change the New function name to "NewSecureTunnel".